web/client/examples/viewer/reducers/fullScreen.js

```diff
@@ -0,0 +1,22 @@
+/**
+ * Copyright 2015, GeoSolutions Sas.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+var {ACTIVATE_BUTTON} = require('../actions/fullScreen');
+
+function fullScreen(state = null, action) {
+    switch (action.type) {
+        case ACTIVATE_BUTTON:
+            return {
+                activeKey: action.activeKey
```

Comment:
here again the reducer react to the TOGGLE_FULLSCREEN with something like: fullscreen : action:fullscreen